cpu/cpu_windows.go

```diff
@@ -15,12 +15,11 @@ import (
 
 type Win32_Processor struct {
 	LoadPercentage            uint16
-	L2CacheSize               uint32
 	Family                    uint16
 	Manufacturer              string
 	Name                      string
 	NumberOfLogicalProcessors uint32
-	ProcessorId               string
+	ProcessorId               *string
```

Comment:
I think it's better to use `string` instead `*string`. The reason is that `*string` may be `nil` and trying to access this value will cause a program to crash. If this is type `string` you can simply test `== ""` to see if it has been specified, but accessing the string normally will always work because it is initialized with the default value.